support-payment-api/src/main/resources/cloud-formation.yaml

```diff
@@ -507,51 +507,70 @@ Resources:
       Statistic: Sum
       TreatMissingData: breaching
 
-  ThreeOrMorePaypalPaymentErrorsIn15Mins:
+  PaypalPaymentError:
       Type: AWS::CloudWatch::Alarm
       Condition: CreateProdResources
       Properties:
         AlarmActions:
         - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:contributions-dev
-        AlarmName: !Sub ${App} ${Stage} Multiple paypal errors in the last fifteen minutes for one-off contributions via the payment-api
+        AlarmName: !Sub ${App} ${Stage} Paypal payment error in the last fifteen minutes for one-off contributions via the payment-api
         MetricName: payment-error
         Namespace: support-payment-api-PROD
         Dimensions:
                 - Name: payment-provider
                   Value: Paypal
         ComparisonOperator: GreaterThanThreshold
-        Threshold: 3
+        Threshold: 1
         Period: 900
```

Comment:
period could change to 60 here so we find out quciker

Reply:
👍